sandy/decay.py

```diff
@@ -184,61 +184,61 @@ def get_branching_ratio(self, with_uncertainty=True):
         Returns
         -------
         `pandas.DataFrame`
-            dataframe with decay mode, branching ratio and associated uncertainty or
-            dataframe with decay mode and branching ratio if with_uncertainty = False
+            dataframe with rtyp, branching ratio and associated uncertainty or
+            dataframe with rtyp and branching ratio if with_uncertainty = False
 
         Examples
         --------
         >>> endf6 = sandy.get_endf6_file("jeff_33", "decay", [942410, 922350])
         >>> rdd = sandy.DecayData.from_endf6(endf6)
         >>> rdd.get_branching_ratio()
-               decay mode          BR         DBR
-        ZAM                                      
-        922350      alpha 1.00000e+00 1.00000e-04
-        922350         SF 7.20000e-11 2.10000e-11
-        942410      alpha 2.44000e-05 0.00000e+00
-        942410       beta 9.99976e-01 0.00000e+00
+               RTYP          BR         DBR
+        ZAM                                
+        922350   40 1.00000e+00 1.00000e-04
+        922350   60 7.20000e-11 2.10000e-11
+        942410   40 2.44000e-05 0.00000e+00
+        942410   10 9.99976e-01 0.00000e+00
 
         >>> endf6 = sandy.get_endf6_file("jeff_33", "decay", [942410, 922350])
         >>> rdd = sandy.DecayData.from_endf6(endf6)
         >>> rdd.get_branching_ratio(False)
-               decay mode          BR
-        ZAM                          
-        922350      alpha 1.00000e+00
-        922350         SF 7.20000e-11
-        942410      alpha 2.44000e-05
-        942410       beta 9.99976e-01
-        
+               RTYP          BR
+        ZAM                    
+        922350   40 1.00000e+00
+        922350   60 7.20000e-11
+        942410   40 2.44000e-05
+        942410   10 9.99976e-01
+
         >>> endf6 = sandy.get_endf6_file("jeff_33", "decay", [942410, 10010, 922350])
         >>> rdd = sandy.DecayData.from_endf6(endf6)
         >>> rdd.get_branching_ratio(False)
-               decay mode          BR
-        ZAM                          
-        922350      alpha 1.00000e+00
-        922350         SF 7.20000e-11
-        942410      alpha 2.44000e-05
-        942410       beta 9.99976e-01
+               RTYP          BR
+        ZAM                    
+        922350   40 1.00000e+00
+        922350   60 7.20000e-11
+        942410   40 2.44000e-05
+        942410   10 9.99976e-01
         """
         br = []
         zam = []
         for z, dic in self.data.items():
             try:
                 for rtyp, dk in dic['decay_modes']:
                     br.append([
-                        rtyp2decaymode(rtyp),
+                        rtyp,
```

Comment:
Why try/except?  Cannot you use `if "decay_modes" in dic` or `if dic["decay_modes"]`?